README.md

```diff
@@ -43,27 +43,27 @@
 
 ### day 1: introduction to azure CLI and powershell scripting
 
-> **conceptual goals**: learn about the az CLI and how to write automated powershell scripts
+> **conceptual goals**: learn about the az CLI and how to write automated powershell scripts and introduction to configurationa nd use cases for window services
```

Comment:
`learn about the az CLI and how to write automated powershell scripts and introduction to configurationa nd use cases for window services` `learn about the az CLI and how to write automated powershell scripts and deploy to an IIS web server on windows server`